src/_settings.scss

```diff
@@ -20,8 +20,9 @@
 // URLs
 $url-statics: '' !default;
 
+@import './settings/box-style';
```

Comment:
Why did you have to move this?

Reply:
I justmoved it to have an alfalbetic order. It may be easier to find a partial scss